dotnet/src/Experimental/Process.Runtime.Dapr/DaprMessage.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Represents a message used in the Dapr runtime.
+/// </summary>
+/// <remarks>
+/// Initializes a new instance of the <see cref="DaprMessage"/> class.
+/// </remarks>
+public record DaprMessage
```

Comment:
This is required to be public because it's used in a Dapr Actor Interface which are required to be public.